modules/release-notes/partials/docs-server-7.6-release-note.adoc

```diff
@@ -97,6 +97,10 @@ This release contains the following fixes.
 | CREATE INDEX statement fails when index key expression matches keyspace.
 | If index key expression matches keyspace, index key expression is no longer converted to SELF.
 
+| https://issues.couchbase.com/browse/MB-60762[MB-60762]
+| ANSI MERGE with DELETE does not always generate an error when the target document matches multiple source documents.
+| ANSI MERGE with DELETE now always generates an error when the target document matches multiple source documents, in compliance with the SQL standard.
+
```

Comment:
This one  (MB-60762) has been reopened & reverted AFAIK, so probably shouldn't be included yet.